packages/core/plugin/src/withSentryIOS.ts

```diff
@@ -79,3 +85,58 @@ export function addSentryWithBundledScriptsToBundleShellScript(script: string):
     (match: string) => `/bin/sh ${SENTRY_REACT_NATIVE_XCODE_PATH} ${match}`,
   );
 }
+
+export function modifyAppDelegate(config: ExpoConfig): ExpoConfig {
+  return withAppDelegate(config, async config => {
+    if (!config.modResults || !config.modResults.path) {
+      warnOnce('Skipping AppDelegate modification because the file does not exist.');
```

Comment:
Same as for Android let's state that the 'RNSentrySDK.start()' won't be added to the AppDelegate.

Reply:
Updated with dce74b25792f6d9b1b1da7e894642b365c9890ed